### DISCUSSION TITLE
First party package for documentation using Swagger

### DISCUSSION THREAD

**Discussion by clicktodev** [laravel/framework#44817] [Ideas]
Laravel has a lot of first party packages that make backend dev extremely fast.
One area I believe should be the next focus is official documentation tooling.

Something like what NestJS [provides](https://docs.nestjs.com/openapi/introduction) with Swagger would be great.

**Comment by shaedrich** [comment]
It would be cool to have this functionality in routes/api.php and/or `FormRequest` class
